site/dat/repo/various.json

```diff
@@ -138,6 +138,38 @@
       }
     }
   },
+  {
+    "id": "PepperBox",
+    "name": "Enhanced Kafka Support with customized schema and SASL support",
+    "description": "This plugin provides an easy way to generate load on Kafka with customized schema, high throughput and with popular security protocols",
+    "screenshotUrl": "",
+    "helpUrl": "https://github.com/GSLabDev/pepper-box",
+    "vendor": "www.gslab.com",
+    "markerClass": "com.gslab.pepper.PepperBoxLoadGenerator.java",
```

Comment:
`markerClass` has to name just fully qualified Java class name, not file name. Looks like `.java` suffix here and in `componentClasses` is not what you intend. Please review.